cicecore/cicedynB/infrastructure/ice_read_write.F90

```diff
@@ -1110,26 +1111,29 @@ subroutine ice_read_nc_xy(fid,  nrec,  varname, work,  diag, &
 ! netCDF file diagnostics:
       integer (kind=int_kind) :: & 
          varid          , & ! variable id
-         status             ! status output from netcdf routines
-!        ndim, nvar,      & ! sizes of netcdf file
-!        id,              & ! dimension index
-!        dimlen             ! dimension size
+         status         , & ! status output from netcdf routines
+         ndims          , & ! number of dimensions
+         dimlen             ! dimension size
+
+      integer (kind=int_kind), dimension(10) :: & 
+         dimids             ! generic size dimids
```

Comment:
It's unfortunate that we have a lot of similar code in this file... I guess a clean-up of that would be for another day ;)